kuma/wiki/jinja2/wiki/document.html

```diff
@@ -383,6 +383,12 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
       </div>
     </div> <!-- ends "main-content" -->
 
+    {% if waffle.switch('newsletter') and waffle.switch('newsletter_article') and waffle.flag('redesign') %}
```

Comment:
These are switches not flags (different part of admin console) but, you should not be seeing the newsletter if you don't have these added and enabled. Maybe they came with your database dump?